services/keyvault/doc.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
```

Comment:
Since `doc.go` files are just a big comment block meant to be picked up by godoc, the convention is to write them in `/*...*/` comments to make them appear as much like a plain-text/README as possible.